arrow-site/docs/docs/fx/README.md

```diff
@@ -11,130 +11,91 @@ Arrow Fx is a functional companion to KotlinX Coroutines augmenting its api with
 
 The library brings purity, referential transparency, and direct imperative syntax to typed FP in Kotlin, and is a fun and easy tool for creating Typed Pure Functional Programs.
 
-# Pure Functions, Side Effects, and Program Execution
-
-## Pure & Referentially Transparent Functions
-
-A pure function is a function that consistently returns the same output when given the same input.
-Pure functions exhibit a deterministic behavior and cause no observable effects externally. We call this property referential transparency.
-
-Referential transparency allows us to reason about the different pieces of our program in isolation.
-
-To create a pure function in Kotlin, let's use the keyword `fun`:
-
-```kotlin:ank:playground
-//sampleStart
-fun helloWorld(): String =
-  "Hello World"
-
-//sampleEnd  
-fun main() {
-  println(helloWorld())
+If you're not familiar yet with Coroutines in Kotlin, it's recommended to first read Kotlin's [Coroutines Guide](https://kotlinlang.org/docs/coroutines-guide.html) on KotlinX Coroutines.
+
+## Arrow Fx Coroutines Overview 
+
+- Quick Start
+  - [Gradle Setup](Gradle-Setup)
+  - [Maven Setup](Maven-setup)
+- Extensions and data types   
+  - [Parallel](parallel/)
+  - [Flow](/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/kotlinx.coroutines.flow.-flow/) 
+  - [Schedule](/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-schedule/)
+  - [Resource](/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-resource/)
+  - [CircuitBreaker](/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-circuit-breaker/)
+  - [Atomic](/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-atomic/)
+  - [STM](/apidocs/arrow-fx-stm/arrow.fx.stm/-s-t-m/index.html)
```

Comment:
Is this link correct? All others end in `somelink/` whereas this ends in `somelink/index.html`

Reply:
Ah ye, the `index.html` can be omitted.
I'll reformat `STM` to be the same as the others.

The STM docs are looking really nice on the webpage 👍
(Arrow Website for next version is broken ATM, we tried to bump Dokka to the new version since the rewrite but we're still blocked :(

Reply:
```suggestion
  - [STM](/apidocs/arrow-fx-stm/arrow.fx.stm/-s-t-m/)
```